crates/hir_ty/src/lower.rs

```diff
@@ -1278,17 +1353,23 @@ pub(crate) fn generic_defaults_recover(
     db: &dyn HirDatabase,
     _cycle: &[String],
     def: &GenericDefId,
-) -> Arc<[Binders<Ty>]> {
+) -> Arc<[Binders<crate::GenericArg>]> {
     let generic_params = generics(db.upcast(), *def);
-
+    // FIXME: this code is not covered in tests. add tests
+    // and remove this never!() or comment why it is unreachable
+    never!();
```

Comment:
please remove these nevers. This code *can* be hit in real-life situations. The FIXME is enough.

Reply:
`never!` is here to notify if some test is added, to remove the comment. But the comment itself is unnecessary, since coverage can be calculated anyway. These just were `todo!()` in my code, and I thought it needs some action. Do you want to keep these FIXMEs?

Reply:
I think the FIXME is good as a reminder to add a test.